docs/user-guide/README.md

```diff
@@ -107,6 +107,7 @@ If you don't have much familiarity with Kubernetes, we recommend you read the fo
   * [Downward API: accessing system configuration from a pod](downward-api.md)
   * [Images and registries](images.md)
   * [Migrating from docker-cli to kubectl](docker-cli-to-kubectl.md)
+  * [Tips and tricks when working with config](config-best-practices.md)
```

Comment:
Please also link to this from docs/user-guide/managing-deployments.md#whats-next